.github/workflows/deploy-server.yml

```
@@ -0,0 +1,130 @@
+    - name: Checkout code
+      uses: actions/checkout@v4
+      
+    - name: Setup Node.js
+      uses: actions/setup-node@v4
+      with:
+        node-version: ${{ env.NODE_VERSION }}
+        cache: 'npm'
+        
+    - name: Install dependencies
+      run: npm ci
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)